### PR TITLE
(PE-31016) Update kitchensink, trapperkeeper, and rbac-client

### PR DESCRIPTION
**Pull request by ekinanp**
This contains the commons-compress bump

Signed-off-by: Enis Inan <enis.inan@puppet.com>

Please add all notable changes to the "Unreleased" section of the CHANGELOG.

### CODE REVIEW

**Comment by ekinanp** [comment]
Hmm, do we need to update tk-version and kitchensink since those also contain commons-compress dep?

**Comment by mwaggett** [comment]
I think bumping those other things too is a good idea @ekinanp

File: project.clj [hunk 1]
```diff
@@ -1,10 +1,10 @@
 (def clj-version "1.10.1")
-(def ks-version "3.1.1")
-(def tk-version "3.1.0")
+(def ks-version "3.1.2")
+(def tk-version "3.1.1")
 (def tk-jetty-version "4.1.3")
```
Comment (ekinanp) [inline project.clj:4]: Think this also needs to be updated but I don't see a new release out yet w/ the commons-compress bump.

Comment (Magisus) [reply project.clj:4]: We shouldn't need to pull in a version with that, the relevant dependencies are all overridden in the consuming projects.

**Comment by mwaggett** [comment]
This needs a changelog update if we still want to merge it.

**Comment by mwaggett** [comment]
@ekinanp Would you mind either closing this or rebasing and updating the changelog? Thanks!

**Comment by mwaggett** [comment]
This PR is extremely stale, so I'm gonna close it. If we end up needing these things, we can re-open.

**Pull request closed without merging**